demisto_sdk/commands/lint/commands_builder.py

```diff
@@ -0,0 +1,212 @@
+# STD python packages
+from pathlib import Path
+from typing import List
+import os
+# Third party packages
+# Local imports
+
+excluded_files = ["CommonServerPython.py", "demistomock.py", "CommonServerUserPython.py", "conftest.py", "venv"]
+
+
+def build_flake8_command(files: List[Path]) -> str:
+    """ Build command for executing flake8 lint check
+        https://flake8.pycqa.org/en/latest/user/invocation.html
+    Args:
+        files(List[Path]): files to execute lint
+
+    Returns:
+        str: flake8 command
+    """
+    max_line_len = 130
+    # Ignoring flake specific errors https://flake8.pycqa.org/en/latest/user/error-codes.html
+    errors_ignoring = ["W293", "W504", "W291", "W605", "F405", "F403", "E999", "W503", "F841", "E302", "C901", "F821",
```

Comment:
Why is this here? We control flake8 ignores and line length in this file: https://github.com/demisto/content/blob/master/tox.ini

Reply:
Didn't noticed that, Changed